wikibase/README.md

```diff
@@ -9,10 +9,16 @@ Automated build.
 
 ### Tags
 
-Image name                                  | Parent image            
-------------------------------------------- | ------------------------
-`wikibase/wikibase` : `latest`, `1.30`      | [mediawiki:1.30](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `legacy`, `1.29`      | [mediawiki:1.29](https://hub.docker.com/_/mediawiki/)
+Image name                                                       | Parent image
+---------------------------------------------------------------- | ------------------------
+`wikibase/wikibase` : `latest`, `1.30`                           | [mediawiki:1.30](https://hub.docker.com/_/mediawiki/)
```

Comment:
This url is also wrong, this links to the mediawiki image, but should link to the wikibase/wikibase image..

Reply:
I think this one is actually right; the *parent* image is mediawiki